metadata.json

```diff
@@ -25,5 +25,6 @@
     { "name": "darin/zypprepo", "version_requirement": ">= 1.0.1 < 2.0.0" },
     { "name": "nanliu/staging", "version_requirement": ">= 1.0.0 < 2.0.0" },
     { "name": "camptocamp/archive", "version_requirement": ">= 0.8.0" }
+    { "name": "logilab/xmldiff", "version_requirement": ">=0.6.10" },
```

Comment:
You're right, I didn't know if the dependencies were only for puppet modules or for every third library. To make this modification a python module is needed, it is called xmldiff (https://pypi.python.org/pypi/xmldiff). What is your suggestion to reflect this needs, modification of the readme ?